src/css/components/header-buttons-mobile.css

```diff
@@ -1,22 +1,34 @@
 .js-overlay-active .mobile-buttons {
   position: fixed;
   top: 1rem;
-  right: 1rem;
+  right: 0;
   z-index: 10;
   left: 0;
-  right: 0;
   background: white;
   height: var(--sp3);
   top: 0;
   padding-right: 1rem;
+  @media (--md) {
+    top: 0;
```

Comment:
You're redeclaring the same value for the top property here.

Reply:
Removed